src/components/login/LoginBox/LoginBox.css

```diff
@@ -0,0 +1,51 @@
+.auth-box-login {
+    padding: 1rem 1.5rem 2.5rem 1.5rem;
+    height: auto;
+    width: 30%;
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px,
+        rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+}
+
+.auth-box-login #h4 {
```

Comment:
you are using same margin at multiple places so you can create one class for that